content/en/blog/_posts/2022-11-XX-seccomp-notifier.md

```diff
@@ -0,0 +1,347 @@
+---
+layout: blog
+title: "Finding suspicious syscalls with the seccomp notifier"
+date: 2022-11-XX
+slug: seccomp-notifier
+---
+
+**Authors:** Sascha Grunert
+
+Debugging software in production is one of the biggest challenges we have to
+face in our containerized environments. Being able to understand the impact of
+the available security options, especially when it comes to configuring our
+deployments, is one of the key aspects to make the default security in
+Kubernetes stronger. We have all those logging, tracing and metrics data already
+at hand, but how do we assemble the information they provide into something
+human readable and actionable?
+
+[Seccomp][seccomp] is one of the standard mechanisms to protect a [Linux][linux]
```

Comment:
No need to hyperlink to Linux; we can assume readers have either heard of it or know of Google.

Reply:
Removed it as suggested